spec/features/support/mock_notification_client.rb

```diff
@@ -10,7 +10,19 @@ def initialize(_unused)
 
     # TODO: remove dependency on test entries in gov-notify.yml
     def send_email(args)
-      self.class.notifications << args[:template_id]
+      self.class.notifications << {
```

Comment:
Just to check my understanding:  
`<<` appends to the end of an array right? 
So `self.class.notifications` is an array of hashes in this method?